lib/mpl_toolkits/axes_grid1/anchored_artists.py

```diff
@@ -52,7 +52,8 @@ def __init__(self, width, height, xdescent, ydescent,
             Font property used as a reference for paddings.
 
         frameon : bool, default: True
-            If True, draw a box around this artists.
+            If True, add a frame and solid background for the artist. The frame's appearance can be changed with the `facecolor` and `edgecolor` kwargs.
```

Comment:
Use 'keyword arguments' instead of 'kwargs'; this needs to be wrapped as well.

Reply:
Updated. Thanks for noting out.

Reply:
It still needs to be word-wrapped at under 80 characters.

Reply:
new line added and seems to fixed the max 80 char issue